src/components/dropdown/DropDown.tsx

```diff
@@ -39,7 +39,7 @@ const DropDown = (props: DropdownProps) => {
                 <StyledLogo width={baseStyle.dropDown.logo.width} src={CoinLogos[item.network]} alt="coinLogo"/>
                 <div>
                   <StyledTitle>{getAbbreviatedAddress(item.address, hiddenAddressGap?.leadingCharLimit, hiddenAddressGap?.trailingCharLimit)}</StyledTitle>
-                  <StyledSubTitle>from: <StyledSpan isRedefined={item.from.startsWith("redefined")}>{item.from.startsWith("redefined") ? "redefined" : item.from}</StyledSpan></StyledSubTitle>
+                  <StyledSubTitle>{item.network} from: <StyledSpan isRedefined={item.from.startsWith("redefined")}>{item.from.startsWith("redefined") ? "redefined" : item.from}</StyledSpan></StyledSubTitle>
```

Comment:
Network is a key
```
{
   key: matic
   chain: Polygon
}
```
Hm... I think, we can add this mapping to connect-assets
So assets will be
```
"matic": {
    name: "Polygon",
    logo: "http://.../assets/polygon.svg",
    symbol: "MATIC",
    type: NetType.EVM,
  }
```

Reply:
https://redefined.atlassian.net/browse/RED-458